packages/devtools-launchpad/src/index.js

```diff
@@ -87,7 +87,7 @@ async function initApp() {
   return { store, actions, LaunchpadApp };
 }
 
-function renderRoot(_React, _ReactDOM, component, _store) {
+function renderRoot(_React, _ReactDOM, component, _store, props) {
```

Comment:
maybe default to an empty object ? not sure if it is risky to pass undefined to createElement though.

Reply:
@nchevobbe we have been passing undefined to `createElement` till now, no? Btw. I tested that and works for me.

Honza

Reply:
true  !